generator/shellgenerator.cpp

```diff
@@ -316,7 +316,7 @@ bool ShellGenerator::functionHasNonConstReferences(const AbstractMetaFunction* f
       QTextStream t(&s);
       t << function->implementingClass()->qualifiedCppName() << "::";
       writeFunctionSignature(t, function, 0, "",
-        Option(ConvertReferenceToPtr | FirstArgIsWrappedObject | IncludeDefaultExpression | OriginalName | ShowStatic | UnderscoreSpaces | ProtectedEnumAsInts));
+        Option(ConvertReferenceToPtr | FirstArgIsWrappedObject | IncludeDefaultExpression | ShowStatic | UnderscoreSpaces | ProtectedEnumAsInts));
```

Comment:
Basically the option said: "Don't use the name that was explicitly set for this method, but use the name from the parsed C++ code." This made a difference for operators, which were set to the name of the Python operators in the AbstractMetaBuilder step. There was code which did this replacement again in the code generator, but this didn't apply to static operators.
I didn't see the purpose of this option, since it also reverted some other renames supplied by the typesystem files, so I removed it. Now the code should behave more straightforward.

Reply:
(I did a before/after comparison of the generated code, and the only difference was that some wrapper methods were not created anymore that are also provided by the super class, and this was because I removed the previously ignored rename entries.)